src/main.go

```diff
@@ -34,6 +34,7 @@ var Config struct {
 	SequencerAddress            string `mapstructure:"sequencer-address"`
 	KeyperSetManagerAddress     string `mapstructure:"keyperset-manager-address"`
 	DelayInSeconds              int    `mapstructure:"delay-in-seconds"`
+	ChainGasLimit               uint64 `mapstructure:"chain-gas-limit"`
```

Comment:
I'd rename that parameter to 
```suggestion
	EncryptedGasLimit               uint64 `mapstructure:"encrypted-gas-limit"`
```
that's what it is called in the [spec]() and the [keypers]().

(sorry empty links right now, I'll find them and update here later).

Reply:
Thanks, renamed it to encrypted gas limit.